compose/foundation/foundation/src/commonMain/kotlin/androidx/compose/foundation/gestures/MouseWheelScrollable.kt

```diff
@@ -165,16 +165,29 @@ internal class MouseWheelScrollNode(
      * It's important for tests that attempt to trigger another scroll after a mouse wheel event.
      */
     private suspend fun Channel<MouseWheelScrollDelta>.busyReceive() = coroutineScope {
+        var firstFrameTimeMs: Long? = null
+        var lastFrameTimeMs: Long? = null
         val job = launch {
             while (coroutineContext.isActive) {
-                withFrameNanos { }
+                withFrameMillis { frameTimeMs ->
```

Comment:
Maybe we should use the scroll event time instead? Not sure that the frame time is the right thing to use for calculating scroll velocity

Reply:
There is no common way to extract event time, it's platform specific. Currently it's the only source of time here

Reply:
Ah. it's in `PointerInputChange`. 🤔